src/DMXSerial.cpp

```diff
@@ -23,7 +23,8 @@
 // This is consistent to the Layout of the Arduino DMX Shield http://www.mathertel.de/Arduino/DMXShield.aspx.
 
 // For using the serial port 1 on a Arduino MEGA 2560 board, enable the following DMX_USE_PORT1 definition.
-// #define DMX_USE_PORT1
+// Otherwise, disable back the definition below.
+#define DMX_USE_PORT1
```

Comment:
This change of the default behaviour will break compatibilty of existing implementations.
I like to avoid this.

This is only required on MEGA when using port 1 for communication and should be changed only in your local repository for your purpose.